nazgul/mysql_model.py

```diff
@@ -8,10 +8,12 @@
 
 class MySQLModel:
     def __init__(self, user='root', host='127.0.0.1', password='', db='bouncer'):
+        self._pass = password
         self._user = user
         self._host = host
-        self._pass = password
-        self._db   = db
+        self._db_name = db
+        # TODO: Should I have autocommit always True?
```

Comment:
Since you are calling commit everywhere, it might be worth it to leave autocommit off in case there are more complicated transactions later on.